packages/sdk/src/verbs.ts

```diff
@@ -70,15 +68,11 @@ export class Verbs<THostedWalletProviderType extends HostedProviderType> {
   }
 
   /**
-   * Get the lend provider instance (internal use)
-   * @returns LendProvider instance if configured
-   * @internal
+   * Get the lend provider instance
+   * @returns LendProvider instance if configured, undefined otherwise
    */
-  get lendProviderInstance(): LendProvider {
-    if (!this.lendProvider) {
-      throw new Error('Lend provider not configured')
-    }
-    return this.lendProvider
+  get lendProvider(): LendProvider | undefined {
+    return this._lendProvider
```

Comment:
Also now sure how we want this to work in the end. this currently makes the LendProvider instance accessible via `verbs.lendProvider`.

Reply:
hmm, yeah I think I did something similar with the wallet providers. now that I think about it I wonder if we should keep these private to start in order to minimize the surface of the external api that has to be maintained

Reply:
going to merge as is and we can continue to think about it